src/webob/acceptparse.py

```diff
@@ -686,10 +686,16 @@ def basic_filtering(self, language_tags):
         suggested as a matching scheme for the ``Accept-Language`` header in
         :rfc:`RFC 7231, section 5.3.5 <7231#section-5.3.5>`, and defined in
         :rfc:`RFC 4647, section 3.3.1 <4647#section-3.3.1>`. It filters the
-        tags in the `language_tags` argument, and returns a list of the tags
-        that match the language ranges in the header according to the Basic
-        Filtering matching scheme, in descending order of preference, together
-        with the qvalue of the range each tag matched.
+        tags in the `language_tags` argument and returns the ones that match
+        the header according to the matching scheme. The returned list is a
+        list of (language tag, qvalue) tuples, in descending order of qvalue;
+        if two or more tags have the same qvalue, they are returned in the same
```

Comment:
Let's break this into two sentences to be consistent with the next sentence. IOW, "...qvalue. If..."